client/src/templates/Introduction/components/CertChallenge.js

```diff
@@ -96,7 +125,6 @@ const CertChallenge = ({
         stepVal => typeof stepVal === 'boolean' && stepVal
       ).length + projectsCompleted;
     const numberOfSteps = Object.keys(steps).length;
-
     setCanViewCert(completedCount === numberOfSteps);
```

Comment:
```suggestion
    setCanViewCert(completedCount === numberOfSteps);
```
I know it's out of scope, but `canViewCert` is misleading.  It's really just `hasCompletedRequiredSteps`.